pose_estimation/src/pose_estimation.cpp

```diff
@@ -0,0 +1,70 @@
+#include <pose_estimation/pose_estimation.hpp>
+
+namespace ariitk::pose_estimation {
+
+PoseEstimation::PoseEstimation() {
+	cam_matrix_(0, 0) = 277.191356;
```

Comment:
Please parameterize this, you can't be building this every time you want to change the camera parameters.

Reply:
Do we make a nodehandle here only or make a new function to set camera parameters and then call it in pose_estimation_ros package?

Reply:
I think it will be better if you just take all the camera parameters as Eigen matrices/vectors arguments and directly use them. The code for actually getting them from the parameter server and putting them in matrix form should be done in pose_estimation_ros.cpp.